src/routes/api/commits-from-date/recover-commits-from-dates.ts

```diff
@@ -0,0 +1,63 @@
+import { Request, Response } from "express";
+import { Subscription } from "models/subscription";
+import { Op } from "sequelize";
+import { getLogger } from "config/logger";
+import safeJsonStringify from "safe-json-stringify";
+
+const DEFAULT_BATCH_SIZE = 5000;
+
+export const RecoverCommitsFromDatePost = async (req: Request, res: Response): Promise<void> => {
+
+	const log = getLogger("RecoverCommitsFromDatePost");
+
+	try {
+
+		const startSubscriptionId = Number(req.query.startSubscriptionId) || 0;
+		const batchSize = Number(req.query.batchSize) || DEFAULT_BATCH_SIZE;
```

Comment:
/nit
Do we want a upper limit to batch size to stop people being monsters, 

~~also do we want any validation that the startSubscriptionId is valid...... i guess we are probably ok since it has to be a number and the where caluse will cause it to return nothing.. so thats fine~~